onnxruntime/wasm/api.h

```diff
@@ -222,4 +230,132 @@ int EMSCRIPTEN_KEEPALIVE OrtRun(ort_session_handle_t session,
  * Caller must release the C style string after use by calling OrtFree().
  */
 char* EMSCRIPTEN_KEEPALIVE OrtEndProfiling(ort_session_handle_t session);
+
+// Training API Section
+
+#ifdef ENABLE_TRAINING_APIS
+/**
+ * @brief Load the checkpoint for training.
+ *
+ * @param checkpoint
+ * @param checkpoint_size
+ * @return ort_training_checkpoint_handle_t
```

Comment:
Could you add more documentation? 

For example, `checkpoint_size` is in bytes? And also mention that this loads the checkpoint from a buffer in memory.